yb-voyager/cmd/eventSegmentMover.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
You should move the code from `eventSegmentDeleter.go` and `eventSegmentMover.go` in the `archiveChangesCommand.go`. No need of these many files.